packages/neutrino-preset-node/index.js

```diff
@@ -35,11 +35,13 @@ module.exports = (neutrino) => {
   neutrino.use(compile, {
     include: [neutrino.options.source, neutrino.options.tests],
     babel: {
-      plugins: [require.resolve('babel-plugin-dynamic-import-node')],
+      plugins: [[require.resolve('fast-async'), { spec: true }], require.resolve('babel-plugin-dynamic-import-node')],
```

Comment:
Does using this with `spec: true` put the nodent runtime in every file that uses async?